src/plugins/console/public/application/components/editor_example.tsx

```diff
@@ -30,16 +32,27 @@ export function EditorExample(props: EditorExampleProps) {
   const elemId = `help-example-${props.panel}`;
   const inputId = `help-example-${props.panel}-input`;
 
+  const editorRef = useRef<HTMLDivElement | null>(null);
+  const editorInstanceRef = useRef<senseEditor.CustomAceEditor | null>(null);
```

Comment:
just interesting can we avoid of using `null`. 

do something like:
`useRef<senseEditor.CustomAceEditor>();`